shared/folders/files/paperkey.js

```diff
@@ -75,5 +75,5 @@ export default connect(
       phase: state.unlockFolders.phase,
     }
   },
-  dispatch => bindActionCreators({onBack: navigateUp, checkPaperKey, toPaperKeyInput, onBackFromPaperKey}, dispatch),
+  dispatch => bindActionCreators({onBack: navigateBack, checkPaperKey, toPaperKeyInput, onBackFromPaperKey}, dispatch),
```

Comment:
mmmm. i think we want this to be up and not back. on ios there isn't really a back (it means to go up). i think the real issue is the nav that pushed this screen is likely doing the wrong thing. can we chat about this today